tests/integ/scala/test_window_frame_suite.py

```diff
@@ -283,19 +307,28 @@ def test_range_between_should_accept_numeric_values_only_when_bounded(session):
         df.select(
             min_("value").over(window.range_between(Window.unboundedPreceding, 1))
         ).collect()
-    assert "Cumulative window frame unsupported for function MIN" in str(ex_info)
+        if not local_testing_mode:
```

Comment:
Same issue as above, the error message is confusing and the actual failure is not caused by numeric values.

Reply:
We should rewrite these two tests for both live connection and local testing.